src/i18n/fi.json

```diff
@@ -159,6 +160,10 @@
   },
   "profileInformation": {
     "address": "Osoite",
+    "addressDescription": "Nämä tiedot olet itse lisännyt Helsinki-profiili -palveluun tai muuhun Helsingin kaupungin asiointipalveluun. Tätä osoitetietoa käytämme silloin, kun olet tunnistautunut kevyesti, eli jotain muuta kuin Suomi.fi -tunnistautumista hyödyntäen. Voit halutessasi poistaa tiedot Helsinki-profiilista, jolloin ne poistuvat myös muista asiointipalveluista. Älä poista tietoja, jos sinulla on asiointi kesken.",
+    "addressDescriptionNoWeakAddress": "Et ole lisännyt toista osoitetta. Toista osoitetietoa käytämme silloin, kun olet tunnistautunut kevyesti, eli jotain muuta kuin Suomi.fi -tunnistautumista hyödyntäen.",
+    "addressDescriptionNoAddress": "Et ole lisännyt osoitetta. Tätä osoitetietoa käytämme silloin, kun olet tunnistautunut kevyesti, eli jotain muuta kuin Suomi.fi -tunnistautumista hyödyntäen.",
```

Comment:
IMO these explanations are somewhat misleading. What address a service uses is not dependent on the user's (current) authentication method. Basically services can use the addresses in whatever way they want, only depending on what addresses a profile contains.

Reply:
True, but it was discussed that there is no need for deeper explanation.